src/library/dao/playlistdao.h

```diff
@@ -40,6 +40,12 @@ class PlaylistDAO : public QObject, public virtual DAO {
         PLHT_UNKNOWN = -1
     };
 
+    enum class AutoDJSendLoc {
+        TOP,
+        BOTTOM,
+        REPLACE,
+    };
+
```

Comment:
Is this enum defined in the right place? Should it be scoped? It leads to e.g. `PlaylistDAO::AutoDJSendLoc::BOTTOM`, which seems a little ugly.